CodeMaid/UI/SelectionBindingTextBox.cs

```diff
@@ -0,0 +1,105 @@
+﻿namespace SteveCadwallader.CodeMaid.UI
```

Comment:
Based on the comment above, I think this class can go away and the Options dialog can be reverted back to its previous behavior.